### ISSUE TITLE
Automatically load new posts in feeds

### ISSUE THREAD

**Issue by xPaw** [bluesky-social/social-app#6383]
### Describe the Feature

There's code to poll for new posts every minute which shows a button to load new posts, but I would like an option to just show new posts as they come so I can simply have bsky open on a second screen.

One solution would be to only load new posts if the page is currently scrolled to the top.

Ideally it would use the firehose instead of polling for updates every minute.

**Comment by SpriterGors** [comment]
Perhaps have an optional toggle that allows you to choose whether you want the feed to load automatically or not (default) could be interesting, gives the user more agency on how it should operate

**Comment by IPv6** [comment]
Oh no, not as default option at least. This is so annoying when you want to interact with next post in feed later - but it flushed away with unwanted autoupdate. imho current depiction that something new present without forcing it is ideal

**Comment by SpriterGors** [comment]
I agree, the default should be the current method, with the autoupdate being an optional feature. I hate it when the feed autouploads and I can't find the posts again
